src/components/AccountDetails/index.tsx

```diff
@@ -28,6 +28,7 @@ const HeaderRow = styled.div`
 
 const UpperSection = styled.div`
   position: relative;
+  background-color: ${({ theme }) => transparentize(0.55, theme.bg2)};
```

Comment:
Not sure if this should be there. The modal component has its own blurred and transparentized background, and I think adding this makes it just less transparent, no? What happens if `background-color: transparent` is used instead?

Reply:
I think it's better to remove the background of the modal components since the background-colors are different in different modals @luzzif

Reply:
instead we can set the individual transparency and background color to each one

Reply:
Oh yeah. I didn't see that the modal background changed since I last saw Figma. You're right.